src/routes/chains/chains.controller.spec.ts

```diff
@@ -281,6 +284,31 @@ describe('Chains Controller (Unit)', () => {
         },
       });
     });
+
+    it('Failure: received data is not valid', async () => {
```

Comment:
This was previously in the suite but [removed](https://github.com/safe-global/safe-client-gateway/pull/1871#discussion_r1740519879).